### PR TITLE
Fix :Switch from travis-ci.org to GitHub Actions CI and setup semantic-rel…

### LINKED ISSUE #47: Switch from travis-ci.org to GitHub Actions CI and setup semantic-release

### PR DESCRIPTION
**Pull request by Gabriel-403**
fix :  #47

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,20 +23,24 @@
...
     "jest": "^26.0.1",
     "lint-staged": "^10.2.9",
     "mysql2": "^2.1.0",
     "prettier": "^2.0.5",
-    "rimraf": "^3.0.2",
     "ts-jest": "^26.1.0",
     "tslint": "^6.1.2",
-    "typescript": "^3.9.5"
+    "typescript": "^3.9.5",
+    "casbin": "^5.0.3"
```
Comment (hsluoyz) [inline package.json:38]: Why?

**Review by hsluoyz — CHANGES_REQUESTED** [review]

**Review by zhmushan — APPROVED** [review]
LGTM - Moving casbin to peer deps has nothing to do with this topic, but I think the right user should have installed both libraries at the same time

File: package.json (hunk 1 above)
Comment (Zxilly) [reply package.json:38]: This can reduce the size of `node_modules`. If place it in the dependecies, user will have 2 node-casbin at the same time.

**Review by Zxilly — APPROVED** [review]
lgtm

File: package.json (hunk 1 above)
Comment (hsluoyz) [reply package.json:38]: @Gabriel-403 @Zxilly why don't them follow the alphabet order?

**Review by hsluoyz — APPROVED** [review]

**Comment by hsluoyz** [comment]
@Gabriel-403 still failing:

https://github.com/node-casbin/sequelize-adapter/runs/3087250561?check_suite_focus=true

image

**Pull request merged**